dts/arm/silabs/efm32pg12b.dtsi

```diff
@@ -10,6 +10,11 @@
 #include "gpio_gecko.h"
 
 / {
+	aliases {
```

Comment:
These aliases are not sufficient to make the watchdog test run. I had to change the source code of the test:

~~~diff
diff --git a/tests/drivers/watchdog/wdt_basic_api/src/test_wdt.c b/tests/drivers/watchdog/wdt_basic_api/src/test_wdt.c
index d2ffb38f3f..9bfe29a8d7 100644
--- a/tests/drivers/watchdog/wdt_basic_api/src/test_wdt.c
+++ b/tests/drivers/watchdog/wdt_basic_api/src/test_wdt.c
@@ -68,7 +68,7 @@
 #ifdef CONFIG_WWDG_STM32
 #define WDT_DEV_NAME DT_WWDT_0_NAME
 #else
-#define WDT_DEV_NAME DT_WDT_0_NAME
+#define WDT_DEV_NAME DT_ALIAS_WATCHDOG_0_LABEL
 #endif
 #endif
~~~
@galak What is the proper fix for that?

Reply:
I think @galak first needs to have PR [#20321](https://github.com/zephyrproject-rtos/zephyr/pull/20321) merged in.

Reply:
Yes, i missed that.